### PR TITLE
Add synchronous api

### PR DESCRIPTION
**Pull request by kordano**
based on #34

### CODE REVIEW

File: .calva/output-window/output.calva-repl [hunk 1]
```diff
@@ -0,0 +1,196 @@
+; This is the Calva evaluation results output window.
```
Comment (whilo) [inline .calva/output-window/output.calva-repl]: Can you remove the .calva output?

**Review by whilo — CHANGES_REQUESTED** [review]
Looks good except for the calva file.

File: .calva/output-window/output.calva-repl (hunk 1 above)
Comment (jsmassa) [reply .calva/output-window/output.calva-repl]: It's being ignored now

**Review by whilo — APPROVED** [review]

**Review by whilo — APPROVED** [review]

**Comment by whilo** [comment]
@jsmassa Sorry for rerequesting your review, it was an accident while I tried to resolve my conversation. It looks good to me now!

**Pull request merged**